src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -14,13 +14,21 @@ import {useHistory} from "shared/utils/hooks/useHistory";
 import {Draggable} from "shared/components/DragDroppable/Draggable";
 import {DragDropHandlers} from "shared/components/DragDroppable/DragDropHandlers";
 
+import {IC} from "digital/models/ioobjects";
+import {ICData} from "digital/models/ioobjects";
```

Comment:
digital should **never** be imported to shared or core, shared is _shared_ between analog and digital, so when analog uses this, it shouldn't have to know what digital is